web/libs/editor/tests/integration/data/lmm/accept_suggestions.ts

```diff
@@ -1,64 +0,0 @@
-import { LabelStudio } from "@humansignal/frontend-test/helpers/LSF";
```

Comment:
strange, i thought this would get picked up by our tests being in the `tests/integration` directory.
weirder still non of the other ones got picked up for deletion

Reply:
The list I received was lengthy. I removed these files because they contained constants that were not in use. Therefore, the entire file can be deleted.